### ISSUE TITLE
Discussion: Import and Export Rainbow Models

### ISSUE THREAD

**Issue by mancoast** [unixpickle/anyrl-py#16]
Greetings,

I am looking into the usage of this library for a project.  Specifically, I seek the ability to export a pre-trained model from the training set when using the Rainbow for use on another (unknown) test set.

Looking forward to your wisdom.

Thanks,
Coast

**Comment by mancoast** [comment]
Thanks!
